client/js/nonprofits/donate/info-step.js

```diff
@@ -38,8 +37,14 @@ function init(donation$, parentState) {
   state.showDedicationForm$ = flyd.map(()=> false, state.submitDedication$)
 
   // Save donor supporter record
-  state.supporterFields = supporterFields.init({required: {email: true}}, parentState.params$)
-  state.savedSupp$ = flyd.flatMap(postSupporter , flyd.map(formatFormData, state.submitSupporter$))
+  let formData$ = flyd.map(formatFormData, state.submitSupporter$)
```

Comment:
Does the `formData$` variable ever change? If not, make it `const`. `const` assists Javascript engines in their optimization.